backend/amd64/emit.mlp

```diff
@@ -1389,13 +1389,17 @@ let emit_probe_handler_wrapper p =
   cfi_endproc ();
   emit_function_type_and_size wrap_label
 
-let emit_probe_notes0 probes =
-  begin match system with
+let is_macosx_system =
```

Comment:
Aren't there three cases, though? as in:
- Linux: currently supported;
- macOS: eventually supported;
- other platforms: not supported.

Reply:
Here is a slightly improved version that checks for unsupported systems explicitly. 
The supported list matches configure, but we haven't tested it on all the different systems that are supposed to be supported and probes_lib probably won't work everywhere "out of the box".